README.md

```diff
@@ -8,17 +8,25 @@ PokeReader is a 3gx plugin that allows viewing information about 3ds Pokemon gam
 
 This information can be used to RNG shiny and high IV Pokemon, similar to lua scripts on emulators.
 
+## Commands
+- X+Y to Lock/Unlock
+- D-pad keys to navigate when unlocked (right & left to get in and out of the different tabs)
+- Start + Select to pause game and enable manual frame advancing
+- Select to advance Main RNG frames once
```

Comment:
We should rephrase this to `Select to advance one frame` for a few reasons:
- RNG frames aren't a real thing - it's a misconception by the community
- This button doesn't do anything with the RNG, just the video frame
- The RNG could advance more than once
- It's possible this won't advance the RNG

"RNG frames" aren't a real thing, so I try to avoid using that phrase when possible so we move the community away from it.  Instead, we have "RNG advances" and "video frames".